types/three/src/Three.Legacy.d.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
What's this?

Reply:
Copied directly from the original library.

Reply:
TL;DR; it should not be empty, but IMO that's OK for now

@joshuaellis
Should it contain empty types resassembling current library types? If I understand it was added to provide feedback, warnings or errors when using legacy objects:
https://github.com/mrdoob/three.js/blob/945ecfd7ea53aebf1d43e8b29ca6e48e084609be/src/Three.Legacy.js

Reply:
@peterblazejewicz, the way the repo is composed, those are extending classes/functions elsewhere in the repo, for example, `Uniform` the `dynamic` property is written in the Legacy file, but the type is written [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50930/files#diff-ee11c4160e1f7b605d53dec6f6575c2a7fa90e31ab41052a720bb511704e7f38R15).